modules/passkey/README.md

```diff
@@ -111,6 +131,10 @@ npx hardhat --network <network> etherscan-verify
 npx hardhat --network <network> local-verify
 ```
 
+### Compiler settings
```

Comment:
I think should we also mention here why optimiser is enabled as follows:

After careful consideration, we decided to enable the optimizer for the following reasons:

- The most critical functionality, such as signature checks is handled by the verifier contract.
- The entrypoint contract uses the optimizer.

Reply:
> The most critical functionality, such as signature checks is handled by the verifier contract.

Does the verifier use an optimiser? Do we control this setting? If yes, then the argument doesn't hold.

> The entrypoint contract uses the optimizer.

I don't think the entrypoint contract is relevant for this because it's possible to use some of the passkey contracts without 4337

In general, I do think we should elaborate on our compiler settings 👍

Reply:
> Does the verifier use an optimiser?
Its an external contract. So, not defined if optimiser is enabled.
> Do we control this setting?
Nope

> I don't think the entrypoint contract is relevant for this because it's possible to use some of the passkey contracts without 4337
Will remove it

Reply:
As verifier contract can be any contract and is not fixed, also as `entrypoint` is optional here, the statements do not justify purpose of optimiser.